scripts/display-travis-status.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+const debug = require('debug')('display_travis_status')
+const pollTravis = require('../lib/pollTravis')
+
+module.exports = function (app) {
+  app.on('pull_request.opened', (event) => {
+    const owner = event.repository.owner.login
+    const repo = event.repository.name
+    if (repo !== 'nodejs.org') return
```

Comment:
> The bot will be used on repos that do not have builds (like  TSC ) or do not use Travis (like  node ).

Ahhhh good point! Atleast allow citgm and readable-stream then. For later we could check if the repo in question uses Travis, but I'm fine with adding that later in another PR.